### PR TITLE
Fix/fb login

### PR DESCRIPTION
**Pull request by nicofh**

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -258,8 +258,12 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :facebook, '241163829759918', '23c20a5525d87e610402a241386b537e', callback_url: 'http://localhost:3000/users/auth/facebook/callback'
-
+  if Rails.env == 'development' || Rails.env == 'test'
+    config.omniauth :facebook, '241163829759918', '23c20a5525d87e610402a241386b537e', callback_url: 'http://localhost:3000/users/auth/facebook/callback'
```
Comment (gabrielrp) [inline config/initializers/devise.rb]: Move the client id, secret and callback url to a environment variable. That way you don't need to check for the environment and, the most important, you are not uploading sensible data to the repo.

**Review by fedeagripa — COMMENTED** [review]
It will be nice to read the uris from an env variable

**Review by gabrielrp — COMMENTED** [review]
Instead of creating commits for each fix or change you want to add, use the `git commit --amend` command. That way you don't end up having a lot of "Fix X" commits that makes the logs less clear. 
Take a look to https://github.com/rootstrap/tech-guides/blob/master/git/commits.md

**Review by gabrielrp — APPROVED** [review]

**Comment by letiesperon** [comment]
Try to add a description to your PRs, specially if they are bug fixes. This helps future developers to understand why you did things that way and to track down changes more easily.

**Review by letiesperon — APPROVED** [review]

**Pull request merged**